### ISSUE TITLE
Check wether build is bitwise reproducible

### ISSUE THREAD

**Issue by captnfab** [pianobooster/PianoBooster#16]

**Comment by pianobooster** [comment]
I don't know what this issue is about @captnfab  please can you close this or provide more information it is important.

**Comment by trebmuh** [comment]
https://reproducible-builds.org/ probably?

**Comment by captnfab** [comment]
Yeah, but I'd say it's more of a Debian preoccupation. You can discard this issue.

**Comment by trebmuh** [comment]
That isn't only Debian at all. See https://reproducible-builds.org/who/.

**Comment by pianobooster** [comment]
"You can discard this issue."

**Comment by trebmuh** [comment]
That is unfortunate @pianobooster The reproducible build effort is a great thing and usually doesn't require a lot of code change. It's all about securing the software and the users. Could you please at least explain why you closed this thread rather than closing it with such a short uninformative message?

**Comment by pianobooster** [comment]
Re-opened. @trebmuh  could I assign this one to you? My previous message quoted a reply from captnfab when I asked him if I could this issue. As a general rule I prefer not not leaving old issues open that there are no plans to work on it. This issue is probably not needed while i am building the executable on my local machine but would be needed if perhaps got a Continuous Integration server doing the builds.

**Comment by trebmuh** [comment]
IMHO, this issue is important if the plan is to get PB in the distros official repo. Because distros are heading towards wanting the builds to be reproducible. [Arch, Debian, Fedora, Guix, Nixos, Opensuse, some BSD, and a few other linux distros and projects are moving forward this goal](https://reproducible-builds.org/who/).

I'd love to be able to help PB (and other project I'm working with) but I've got close to 0 knowledge in programming. That said, on my next PB build, I'll check what stuff are needed to move on this reproducible build target.
